test/acceptance/features/steps/app.py

```diff
@@ -10,11 +10,13 @@ class App:
     namespace = ""
     app_image = ""
     route_url = ""
+    bindingRoot = ""
 
-    def __init__(self, name, namespace, app_image):
+    def __init__(self, name, namespace, app_image, bindingRoot):
```

Comment:
Move `bindingRoot` to `install` method and not set the env. var  if it is `None` (`None` is the default value)